packages/core/src/routes/experience/classes/verifications/social-verification.ts

```diff
@@ -102,11 +115,21 @@ export class SocialVerification implements IdentifierVerificationRecord<Verifica
     tenantContext: TenantContext,
     { state, redirectUri }: SocialAuthorizationUrlPayload
   ) {
-    return createSocialAuthorizationUrl(ctx, tenantContext, {
-      connectorId: this.connectorId,
-      state,
-      redirectUri,
-    });
+    return createSocialAuthorizationUrl(
+      ctx,
+      tenantContext,
+      {
+        connectorId: this.connectorId,
+        state,
+        redirectUri,
+      },
+      {
+        setSession: async (connectorSession) => {
+          this.connectorSession = connectorSession;
+        },
+        jti: this.id,
+      }
```

Comment:
Would it be better to create separate social `createSocialAuthorizationUrl` and `verifySocialIdentity` methods without the dependency on the interaction session?  So we can keep the flow isolated.